CONTRIBUTING.md

```diff
@@ -504,11 +538,15 @@ def test_my_plotting_case():
 
 ### Documentation
 
+Most documentation sources are in the `doc` folder. The documentation are
+written in [reStructureText](https://docutils.sourceforge.io/rst.html) and
+built by [sphinx](http://www.sphinx-doc.org/). Please refer to
+[reStructuredText Cheatsheet](https://docs.generic-mapping-tools.org/latest/rst-cheatsheet.html)
+if you are new to reStructureText.
```

Comment:
They are related to https://github.com/GenericMappingTools/pygmt/pull/1119#issuecomment-810847026.